.circleci/smoke.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMPDIR="$(mktemp -d 2>/dev/null || mktemp -d -t 'apiejs-smoketest')"
```

Comment:
Just to mention, this is to count for platform differences (BSD mktemp vs GNU mktemp). *one* of these will paths will pass to create the tmp directory.

Reply:
That's maybe something which would be worth also as an inline comment right above that line.